client/src/views/CellView.tsx

```diff
@@ -41,9 +43,8 @@ const CellView = () => {
   const Confirm = useConfirm();
   const Alert = useAlert();
   const { authing, user, updateData } = useAuth();
-
   const { data: logs } = useStateAsync([], Server.Log.list, bruteName || '');
-
+  const bruteList : string[] | undefined = user?.brutes.map((bruteDetails) => bruteDetails.name);
```

Comment:
Same thing as previously, don't define variable that you don't use outside handlers